ansible/vars/dev-vagrant.yml

```diff
@@ -30,3 +30,7 @@ site_user_email_confirmation_enabled: false
 
 # Algolia
 site_algolia_search_prefix: 'dev_'
+
+# Notification mailer
+notification_from_address: "stage.librecores.org"
+notification_from_name: "LibreCores Notifications"
```

Comment:
You also need to update production.yml and staging.yml.  And use the right domain for each of them, with librecores.devel being the domain for dev-vagrant.